<!DOCTYPE HTML>
<html>
<head>
<title>Home</title>
<link href="css/bootstrap.css" rel="stylesheet" type="text/css" media="all">
<!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
<!-- Custom Theme files -->
<link href="css/style.css" rel="stylesheet" type="text/css" media="all"/>
<!-- Custom Theme files -->
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="keywords" content="" />
<!-- <script type="application/x-javascript"> addEventListener("load", function() { setTimeout(hideURLbar, 0); }, false); function hideURLbar(){ window.scrollTo(0,1); } </script> -->
<!--Google Fonts-->
<link href='http://fonts.googleapis.com/css?family=Roboto:400,500,700' rel='stylesheet' type='text/css'>
<link href='http://fonts.googleapis.com/css?family=Ubuntu+Condensed' rel='stylesheet' type='text/css'>
<!--google fonts-->
<script src="js/jquery-1.11.0.min.js"></script>
<script src="js/bootstrap.min.js"></script>
<style>
	.ctitle{
		font-size: 36px;
		font-weight: bold;
		margin: 30px 0;
    	 color: #F7B400;;
		font-family: 'Ubuntu Condensed', sans-serif;
		text-align: center;
	}
</style>
</head>
<body>
<!--header-top start here-->
<div class="top-header">
	<div class="container">
		<div class="top-header-main">
			<div class="col-md-8 header-address">
	
			</div>
			<div class="col-md-4 top-social">
	
			</div>
			
		  <div class="clearfix"> </div>
		</div>
	</div>
</div>
<!--header-top end here-->
<!--header start here-->
	<!-- NAVBAR
		================================================== -->
<div class="header w3l">
	<div class="fixed-header">	

		    <div class="navbar-wrapper">
		      <div class="container">
		        <nav class="navbar navbar-inverse navbar-static-top">
		             <div class="navbar-header">
			              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
			                <span class="sr-only">Toggle navigation</span>
			                <span class="icon-bar"></span>
			                <span class="icon-bar"></span>
			                <span class="icon-bar"></span>
			              </button>
			              <div class="logo">
			                   <h1><a class="navbar-brand" href="index.html">Au Lab</a></h1>
			              </div>
			          </div>
		            <div id="navbar" class="navbar-collapse collapse">
		            <nav class="cl-effect-16" id="cl-effect-16">
		              <ul class="nav navbar-nav">
		               <li><a class="active" href="index.html" data-hover="Home">Home</a></li>
		                <li><a href="people.html" data-hover="People">People</a></li>
						<li><a href="research.html" data-hover="Research">Research</a></li>
						<li><a href="publications.html" data-hover="Publications">Publications</a></li>
						<li><a href="software.html" data-hover="Software">Software</a></li>
						<li><a href="news.html" data-hover="News">News</a></li>
						<li><a href="contact.html" data-hover="Contact">Contact</a></li>
                        <li><a href="file.html" data-hover="File">File</a></li>
		              </ul>
		            </nav>

		            </div>
		            <div class="clearfix"> </div>
		             </nav>
		          </div>
		           <div class="clearfix"> </div>
		    </div>
	 </div>
</div>
<!--header end here-->
<!--banner  start hwew-->
<div class="banner">
	<div class="container">
		<div class="banner-main">
            <br/>
            <br/>
            <br/>
			<h2>Welcom To Au Lab!</h2>
			<span class="ban-line"> </span>	
			<p>THE STATISTICAL BIOINFORMATICS & COMPUTATIONAL LAB <BR>@ THE OHIO STATE UNIVERSITY</p>
		    <a href="contact.html">Get in Touch</a>
		</div>
    </div>
</div>


<!--BANNER END HERE-->


</br>
<div class="gallery agile">
	<div class="container">
		<div class="gallery-main">
			<div class="gallery-top fadeInRightBig" data-wow-delay="0.1s">
            </br>
				<h2>News</h2>
			</div>
            
			<div class="gallery-bottom wow zoomIn" data-wow-delay="0.1s">
                
                
                <div class="view view-seventh"><h4>Publication on Jan 17th, 2020: Performance difference of graph-based and alignment-based hybrid error correction methods for error-prone long reads.</h4></br>
			   	   <div class="project-eff">
	                    <div id="nivo-lightbox-demo">  <a href="news.html"><span class="rollover1"> </span>
								<img class="img-responsive" src="images/x.png" alt="">
		                    <div class="mask">
                            <h3>Performance difference of graph-based and alignment-based hybrid error correction methods for error-prone long reads.</h3>
		                    </div></a>
	                   </div>
                   </div>
                </div>
                
                
                <div class="view view-seventh"><h4>March, 2020: Congrats! Professor Au joined the editorial boards of the premier journals of genomics and bioinformatics.
                </h4>
                <div class="project-eff">
                <div id="nivo-lightbox-demo">
                <a href="news.html" ><span class="rollover1"> </span>
                <img class="img-responsive" src="images/news_cover.png" width = 340 alt="">
                <div class="mask">
                <h3>March, 2020: Congrats! Professor Au joined the editorial boards of the premier journals of genomics and bioinformatics.</h3>
                </div></a>
                </div>
                </div>
				</div>
				

				<div class="view view-seventh"><h4>Publication on Jun 6th, 2020: A network-based computational framework to predict and differentiate functions for gene isoforms using exon-level expression data.</h4></br>
					<div class="project-eff">
					  <div id="nivo-lightbox-demo">  <a href="news.html" ><span class="rollover1"> </span>
							  <img  class="center-block" src="images/METHODS.jpg" width = 300 alt="" >
						  <div class="mask">
							  <h3>A network-based computational framework to predict and differentiate functions for gene isoforms using exon-level expression data.</h3>
						  </div></a>
					 </div>
				 </div>
			  </div>
                
		</div>
	</div>
</div>




<div class="container">
	<div class="row">
    <div class="gallery-top fadeInRightBig" data-wow-delay="0.3s">
    </br>
    </br>
    <h2>Open Positions</h2>
    </div>
            <b>The Au Lab is seeking candidates for two postdoctoral research scholar positions to work on sequencing analyses:</b>
            <br>
            <br>
            <li><b>Postdoctoral Research Scholar in bioinformatics method development:</b></li>
              We are looking for a postdoctoral scholar to develop statistical and computational/algorithmic methods for high-throughput sequencing data analyses, such as transcriptome and epigenetics using PacBio and Oxford Nanopore Technologies. Background on mathematics, statistics, computer science or degree in a related field is desirable.
            <br>
            <br>
            <li><b>Postdoctoral Research Scholar in computational biology</b></li>
            We are looking for a postdoctoral scholar in biology background. Our lab performs world leading science in development process and cancer research. Our team focuses on making meaningful advances to understand development process and cancer, using cutting edge technologies (third generation sequencing technology) to create authentic models and gain novel biological insight in transcriptome and epigenetics level. A typical candidate would have skills in including but not limited to implementing basic in-house bioinformatics tools, collaborating with statistician and computational biologists to elucidate biological discovery and mining out biological insights. Background on biology, computational biology or bioinformatics is desirable.
            
            
            <p>Strong ability in statistical analysis or computer programming (python/perl, C/C++, R, Matlab) is desirable.</p>
            The postdoctoral research scholars will have the opportunity to collaborate with internationally recognized biomedical scientists at University of Iowa Carver College of Medicine, Stanford University, University of Oxford and Pacific Biosciences.
            
            <br>

            
            <p>
            <br>
            </p>
            
			</div>
		</div>
	</div>
</div>


<div class="news">
<div class="container">
<div class="col-md-6 news-left">
<h3>Lab Arts: Genome Research Cover Drafts</h3>
<section class="accordation_menu">
<p>Single-molecule long-read sequencing reveals the chromatin basis of gene expression. Genome Research. 2019 08; 29(8)1329-1342</p></br>
<div class="col-md-6 new-pic1">
<img src="images/pic1.jpg" alt="" class="img-responsive">
</div>
<div class="col-md-6 new-pic1">
<img src="images/pic3.jpg" alt="" class="img-responsive">
</div>
<div class="col-md-6 new-pic1">
<img src="images/pic5.jpg" alt="" class="img-responsive">
</div>
<div class="col-md-6 new-pic1">
<img src="images/pic4.jpg" alt="" class="img-responsive">
</div>
<div class="col-md-6 new-pic1">
<img src="images/pic6.jpg" alt="" class="img-responsive">
</div>
<div class="col-md-6 new-pic1">
<img src="images/pic2.jpg" alt="" class="img-responsive">
</div>

</div>

<div class="col-md-6 we-choose-rit">
<h3> Thoughts behind Arts</h3>
</br>
</br>
<div class="menu_vertical">
<section class="accordation_menu">
</br>
<h3>Cats and Yarns: </h3>
</br>
<p> The DNA strands are wrapped around histone protein cores like "yarn balls", which form nucleosomes. Some DNA regions (threads) are folded as “close chromatin” while loose threads are “open chromatin”.  The previous techniques can only identify these epigenetics statuses at bulk level while our new method “MeSMLR-seq” makes the first sharp-eyed observation as cat: nucleosome positioning and chromatin openness at each single DNA molecules (threads and balls with different colors) can be seen. In addition, cat likes yarn balls.
</br></br>
<h3>Railway, Mountains, and Valleys: </h3>
</br>
<p> Railway represents the double-stranded DNA molecules. Different railways stretching around mountains (i.e., nucleosomes) indicate different DNA molecules. The road lights represent the exogenous 5mC at GpC sites introduced by our MeSMLR-seq protocol, which label and show the linker DNA fragments between nucleosomes (mountains) and the open chromatin regions (town). The town with many road lights shows vigorous of human activities. i.e., the open chromatin regions are active for many biological processes. For example, the buildings/houses indicate transcription factors binding to open promotors for gene transcriptions. The dark mountain areas represent the regions occupied by nucleosomes, with less biological events.
</br>
</br>
<h3>Temples, Moutains and Clouds:</h3></br>
<p>This is similar with “railway and mountain”, where cloud represents DNA molecules and pine tree represents the exogenous 5mC at GpC sites introduced by MeSMLR-seq. It could be in a style of Chinese painting.
</p>

</div>
<div>
</div>
<div>




</div>
</section>
</div>
</div>
</div>

</div>





<!--copy rights start here-->
<div class="copy-rights">
	<div class="container">
		<div class="copy-rights-main">
			
        
            <li>&copy; <a href=" http://www.augroup.org">Kin Fai Au</a>. All Rights Reserved.</li>
            <li><a href="https://www.osu.edu/">The Ohio State University</a>, OH 43210.</li>
            
		</div>
	</div>
</div>
<!--copy rights end here-->
</body>
</html>

      	 
