<!DOCTYPE HTML>
<html>
	<head>
		<title>News</title>
		<link href="css/bootstrap.css" rel="stylesheet" type="text/css" media="all">
		<!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
		<!-- Custom Theme files -->
		<link href="css/style.css" rel="stylesheet" type="text/css" media="all" />
		<!-- Custom Theme files -->
		<meta name="viewport" content="width=device-width, initial-scale=1">
		<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
		<meta name="keywords" content="" />
		<script type="application/x-javascript">
			addEventListener("load", function() {
				setTimeout(hideURLbar, 0);
			}, false);

			function hideURLbar() {
				window.scrollTo(0, 1);
			}
		</script>
		<!--Google Fonts-->
		<link href='http://fonts.googleapis.com/css?family=Roboto:400,500,700' rel='stylesheet' type='text/css'>
		<link href='http://fonts.googleapis.com/css?family=Ubuntu+Condensed' rel='stylesheet' type='text/css'>
		<!--google fonts-->
		<script src="js/jquery-1.11.0.min.js"></script>
		<script src="js/bootstrap.min.js"></script>
		<!-- Global site tag (gtag.js) - Google Analytics -->
		<script async src="https://www.googletagmanager.com/gtag/js?id=UA-179571618-1"></script>
		<script>
  			window.dataLayer = window.dataLayer || [];
  			function gtag(){dataLayer.push(arguments);}
  			gtag('js', new Date());
  			gtag('config', 'UA-179571618-1');
		</script>
        <style>
			.title {
				font-size: 36px;
				font-weight: bold;
				margin: 30px 0;
				 color: #F7B400;;
				font-family: 'Ubuntu Condensed', sans-serif;
			
			}
			.second_title
			{
				font-size: 24px;
				font-weight: bold;
				margin:30px 30px 30px 0;
				
			}
			.Context{
				margin:30px  0;
			}
		</style>
	</head>
	<body>
		<!--header-top start here-->
		<div class="top-header">
			<div class="container">
				<div class="top-header-main">
					<div class="col-md-8 header-address">

					</div>
					<div class="col-md-4 top-social">

					</div>

					<div class="clearfix"> </div>
				</div>
			</div>
		</div>
		<!--header-top end here-->
		<!--header start here-->
		<!-- NAVBAR
		================================================== -->
		<div class="header w3l">
			<div class="fixed-header">

				<div class="navbar-wrapper">
					<div class="container">
						<nav class="navbar navbar-inverse navbar-static-top">
							<div class="navbar-header">
								<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar"
								 aria-expanded="false" aria-controls="navbar">
									<span class="sr-only">Toggle navigation</span>
									<span class="icon-bar"></span>
									<span class="icon-bar"></span>
									<span class="icon-bar"></span>
								</button>
								<div class="logo">
									<h1><a class="navbar-brand" href="index.html">Au Lab</a></h1>
								</div>
							</div>
							<div id="navbar" class="navbar-collapse collapse">
								<nav class="cl-effect-16" id="cl-effect-16">
								  <ul class="nav navbar-nav">
                                  <li><a class="active" href="index.html" data-hover="Home">Home</a></li>
                                  <li><a href="people.html" data-hover="People">People</a></li>
                                  <li><a href="research.html" data-hover="Research">Research</a></li>
                                  <li><a href="publications.html" data-hover="Publications">Publications</a></li>
                                  <li><a href="software.html" data-hover="Software">Software</a></li>
                                  <li><a href="news.html" data-hover="News">News</a></li>
                                  <li><a href="contact.html" data-hover="Contact">Contact</a></li>
                                  <li><a href="file.html" data-hover="File">File</a></li>
								  <li><a href="archive.html" data-hover="Archive">Archive</a></li>
								  </ul>
								</nav>
								

							</div>
							<div class="clearfix"> </div>
						</nav>
					</div>
					<div class="clearfix"> </div>
				</div>
			</div>
		</div>
		<!--header end here-->
		<!--news start here-->
        <div class="about">
        <div class="container">
        <div class="about-main">
        <div class="about-top">
        <h2>News</h2>
        </div>
        
		
        <div class="row">
			<p class="second_title">Feb 2023: Yunhao and our collaborators’ RNA m6A work was accepted in principle by Nature Structural & Molecular Biology! Congratulations!</p>
			<div class="Context">
				In this work, together with our collaborators Profs. Arne Klungland and John Arne Dahl, Dr. Yanjiao Li (co-first author), we define the RNA N6-methyladenosine (m6A) landscape in mouse oocytes and early embryos using the low-input approach we have developed (another work which is now under review in another Nature sister journal).  We find that m6A is widely deposited in key genes during the maternal-to-zygotic transition (i.e., the stage-specifically expressed transcription factor coding genes, the maternally-inherited transcripts to be degraded post fertilization, the zygotically-activated genes during zygotic genome activation), and the RNAs derived from the stage-specifically activated retrotransposons (e.g., MERVL, MTA).
			</div>
			<img class="img-responsive" width=600 src="images/nsmb_yh_paper.png" alt=""></a>

			<p class="second_title">Feb 2023: We are thrilled to join the Department of Computational Medicine and Bioinformatics at University of Michigan and start a new journey of more exciting researches! 
							<br>We are also recruting talented students and postdoctoral scholars to join our team. For more information, please refer to our homepage and feel free to contact us!</p>
		<p class="second_title">July 2022: Kin Fai is appointed as the Vice Chair of Research at the Department of Biomedical Informatics. Congratulations!</p>
		<p class="second_title">May 2022: Dingjie and Haoran's latest manuscript of LRGASP project was accepted in principle by Nature Methods! Congratulations!</p>
		<div class="Context">								
				<img class="img-responsive" width=600 src="images/LRGASP.png" alt=""></a>
		</div>
		<p class="second_title">Oct. 12th, 2021: The Au Lab receives two NIH R01 grants from National Human Genome Research Institute and National Institute of General Medical Sciences to develop a series of bioinformatics tools to study gene isoforms and transposable elements in human stem cells and germ cells.</p>
		<div class="Context">
			<a href="https://reporter.nih.gov/search/_q4kAuB6EUqzBxZzJlXEcg/projects"><span class="rollover1"> </span>
								
				<img class="img-responsive" src="images/fund_image.png" alt=""></a>
		</div>
		<p class="second_title">Jul. 15th, 2021: The Au Lab receives an NIH R01 grant from NATIONAL HUMAN GENOME RESEARCH INSTITUTE to develop a series of bioinformatics tools to analyze gene isoforms in human stem cells</p>
		<p class="second_title">Jul. 12th, 2021: Congratulation on Haoran's paper accepted by Bioinformatics! </p>
		<div class="Context">
		<b>Real-time mapping of nanopore raw signals.</b><br>
		Zhang, H.*, <b><a>Li, H.*</a></b>,Jain, C., Cheng, H., <b><a>Au, K.F.**</a></b>, Li, H.**, Aluru, S.**
		<br><i>Bioinformatics.</i> 2021.  [<a href="https://doi.org/10.1093/bioinformatics/btab264">Manuscript</a>]<br>
		<b>* These authors contributed equally to this work.</b><br>
		<b>** Co-corresponding author</b>
		</div>
		<p class="second_title">Jul. 1st, 2021: Kin Fai serves a regular member at the NIH Genomics, Computational Biology and Technology Study Section.</p>

		
		<p class="second_title">Dec. 16th, 2020: Congrats! Kin Fai receives the OSU-BMI Award for Research Excellence!</p>

		<p class="second_title">Dec. 15th, 2020: Congratulations to Yunhao, Audrey, Diana and Yuru for their article accepted by Nature Biotechnology!</p>
		<div class="Context">
			<!-- <img src="images/bio.png" width = 150> -->
			<b>Nanopore sequencing technology, bioinformatics and applications. </b><br>
			<b>Wang, Y., Zhao, Y., Bollas, A., Au, K.F.#</b>
			<br>in press,<i> Nature Biotechnology.</i> 2020<br>
			<b># Corresponding author</b>
		</div>
		<div class="Context">
		<b>Single-molecule long-read sequencing reveals a conserved selection mechanism determining intact long RNA and miRNA profiles in sperm.</b><br>
		Sun, Y.H.*, <b><a>Wang, A.*</a></b>,Song, C., Srivastava, R.K., <b><a>Au, K.F.**</a></b>, Li, X.Z.**
		<br><i>Nature Communications.</i> 2020.  [<a href="https://doi.org/10.1101/2020.05.28.122382">Manuscript</a>]<br>
		<b>* These authors contributed equally to this work.</b><br>
		<b>** Co-corresponding author</b>
		</div>

		<p class="second_title">Dec. 16th, 2020: Congrats! Kin Fai receives the OSU-BMI Award for Research Excellence!</p>

		<p class="second_title">Dec. 15th, 2020: Congratulations to Yunhao, Audrey, Diana and Yuru for their article accepted by Nature Biotechnology!</p>
		<div class="Context">
			<!-- <img src="images/bio.png" width = 150> -->
			<b>Nanopore sequencing technology, bioinformatics and applications. </b><br>
			<b>Wang, Y., Zhao, Y., Bollas, A., Au, K.F.#</b>
			<br>in press,<i> Nature Biotechnology.</i> 2020<br>
			<b># Corresponding author</b>
		</div>

		<p class="second_title">Jun 10th, 2020: New Publications.</p>
		<div class="Context">
			<b>A network-based computational framework to predict and differentiate functions for gene isoforms using
				exon-level expression data.</b><br>
			<b>Wang, D.</b>, Zou, X., <b>Au, K.F.#</b>
			<br><i>Methods.</i> 2020. [<a
				href="https://www.sciencedirect.com/science/article/pii/S1046202319302737">Manuscript</a>]<br><b># Corresponding
				author</b>
		</div>
	
        
        <p class="second_title"> March, 2020: Congrats! Professor Au joined the editorial boards of the premier journals of genomics and bioinformatics.</p>
        <div class="Context">
        <img src="images/bio.png" width = 150>
        <b> Kinfai joined the editorial board of <a href="https://genomebiology.biomedcentral.com/about/editorial-board">Genome Biology </a>.</b></br>
        <img src="images/gr.jpg" width = 150>
        <b> Kinfai joined the editorial board of <a href="https://genome.cshlp.org/site/misc/about.xhtml">Genome Research</a>.</b><br>
        </div>
        
        </div>
    
        <div class="row">

        <p class="second_title">Jan 17th, 2020: New Publications.</p>
        <div class="Context">
			<b>Performance difference of graph-based and alignment-based hybrid error correction methods for error-prone
				long reads.</b><br>
			<b>Wang, A., Au, K.F.#</b>
			<br><i>Genome Biology.</i> 2020. [<a
				href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1885-y">Manuscript</a>]
			<br><b># Corresponding author</b>
        </div>

        </div>
    
			<div class="row">

					<p class="second_title">Audrey presents our recent work at the Nanopore Community Meeting 2019.</p>
					<img src="images/dna.jpg" width = 450 />
                    <div class="Context">
						
						Our genome has different levels of organization, and at the nucleosome level, DNA strands wrap around protein cores like beads on a string. Positioning of these nucleosomes and changes in chromatin status play important regulatory roles in gene expression. Our new method, MeSMLR-seq, aims to resolve the long-range dynamics that remain poorly understood at the single-molecule level. Briefly, two examples from the paper in Genome Research are discussed: 1) MeSMLR-seq can resolve differential nucleosome organization principles at the transcription start site of silent versus active genes; and 2) MeSMLR-seq can explore the coupled chromatin status of adjacent genes during transcriptional reprogramming. These methods offer promising utility to study multiple layers of epigenetics simultaneously.[<a href="https://nanoporetech.com/resource-centre/single-molecule-long-read-sequencing-reveals-chromatin-basis-gene-expression-0?ite=17668&ito=1641&itq=acb09df6-2c01-4122-90ec-a5b547959682&itx%5Bidio%5D=624930">Video</a>]
					</div>

			</div>
            
            
			<div class="row">

                <p class="second_title">Jun 14th, 2019: Our latest paper of identifying nucleosome occupancy and chromatin accessibility on single DNA molecule is selected as the cover of the August Issue of Genome Research.</p>
					<img src="images/gr.gif" />
					<div class="Context">

						
						<h3>Single-molecule long-read sequencing reveals the chromatin basis of gene expression. </h3>
                        <br><b>Wang, Y., Wang, A., Liu, Z.</b>, Thurman, A., Powers, L.S., Zou, M., Hefel, A., Li, Y., Zabner, J., <b>Au, K.F.</b><br><i> Genome Res.</i> <b>Cover photo of issue August 2019 </b>. [<a href="https://genome.cshlp.org/content/early/2019/06/14/gr.251116.119.full.pdf+html">Manuscript</a>]
                            <br>
                        <br><td>Combining methyltransferase treatment with nanopore sequencing, Wang et al. developed a new experimental approach “MeSMLR-seq” to map nucleosome occupancy and chromatin accessibility at single long DNA molecules. MeSMLR-seq shows consistent bulk-cell results with existing methods (e.g. MNase-seq and ATAC-seq), but more importantly it brings unprecedented power to capture the single-molecule epigenome. Thus, epigenome heterogeneity can be clearly resolved and subtle but meaningful epigenome differences at single molecules can be discovered. For instance, the MeSMLR-seq data provided the quantitative evidence for the correlation between promoter accessibility and transcription activity. The MeSMLR-seq long reads phased >300 nucleosomes and identified chromatin accessibility of up to 40 adjacent genes simultaneously at single molecules - e.g. the MeSMLR-seq data revealed the coupled chromatin status changes of the adjacent glucose transporter genes responding to the glucose concentration changes. It shows potential to study the association of multiple layers of epigenomes, including base modifications at the same molecules.
                        </td>


				</div>
			</div>
            
            
			<div class="row">

					<p class="second_title">Jun 15th, 2018: Shuhua Fu, Anqi Wang and Andrew Thurman are going to make presentations in ISMB 2018, July 7th, Chicago.</p>
					<div class="Context">
                    <b>Topics:</b>
                    <br>IDP-denovo: <i>de novo</i> transcriptome assembly and isoform annotation by hybrid sequencing. <b>(Shuhua Fu)</b>
                    <br>Theoretical analysis of graph-based and alignment-based hybrid error correction methods for error-prone long reads. <b>(Anqi Wang)</b>
                    <br>Gene isoform abundance quantification with Third Generation transcriptome sequencing. <b>(Andrew Thurman)</b>
					</div>

			</div>
            
            <div class="row">

            <p class="second_title">Feb 23rd, 2018: IDP-denovo paper is published.</p>
            <div class="Context">
            <b>IDP-denovo: <i>de novo</i> transcriptome assembly and isoform annotation by hybrid sequencing.</b>
            <br><b>Fu, S.</b>, <b>Ma Y.</b>, Yao, H., Xu, Z., Chen, S., Song, J., <b>Au, K.F.</b>
            <br><i>Bioinformatics</i> 2018. [<a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty098/4904264">Manuscript</a>]
            </div>

            </div>
			
		</div></div>
	<!--news end here-->
		
		<!--news letter end here-->
		<!--copy rights start here-->
		<div class="copy-rights">
			<div class="container">
				<div class="copy-rights-main">

                    <li>&copy; <a href=" http://www.augroup.org">Kin Fai Au</a>. All Rights Reserved.</li>
                    <li><a href="https://umich.edu/">University of Michigan</a>, MI 48109.</li>

				</div>
			</div>
		</div>
		<!--copy rights end here-->
	</body>
</html>
